### PR TITLE
Removing required

### PR DESCRIPTION
**Pull request by dakota002**

### CODE REVIEW

File: gcn/notices/fermi/gbm/Trigger.example.json [hunk 1]
```diff
@@ -6,12 +6,10 @@
...
   },
   "identifiers": {
-    "instrument_id": "id number",
-    "instrument_id_alternate": "null",
-    "data_archive_page": "null"
-  },
-  "event_info": {
-    "event_id": "Name of the event",
+    "event_id": "id number",
+    "event_id_alternate": "null",
```
Comment (lpsinger) [inline gcn/notices/fermi/gbm/Trigger.example.json:10]: Why the string “null”?

Comment (dakota002) [reply gcn/notices/fermi/gbm/Trigger.example.json:10]: I have no clue! This field is not required, so it should probably just be omitted from the example. I will remove it and test the validation again. Same for the `data_archive_page`

File: gcn/notices/core/Identifiers.schema.json [hunk 2]
```diff
@@ -5,18 +5,25 @@
...
-      "description": "Alternate id number"
+      "description": "Alternate instrument specific ID number"
     },
     "data_archive_page": {
       "type": "string",
       "description": "URL of archived data files"
+    },
+    "event_name": {
+      "type": "string",
+      "description": "Name of the event (ex: GRB 211211A, GW170817, AT2017gfo, SGR 1806-20, etc)"
```
Comment (jracusin) [inline gcn/notices/core/Identifiers.schema.json:22]: as it should appear in the literature

File: gcn/notices/core/Identifiers.schema.json [hunk 3]
```diff
@@ -5,18 +5,25 @@
...
+    },
+    "event_alias": {
```
Comment (jracusin) [inline gcn/notices/core/Identifiers.schema.json:24]: event_name_alternates

File: gcn/notices/core/Identifiers.schema.json [hunk 4]
```diff
@@ -5,18 +5,25 @@
   "title": "Identifiers",
   "description": "Identifiers of trigger and archive",
   "properties": {
-    "instrument_id": {
+    "event_id": {
```
Comment (jracusin) [inline gcn/notices/core/Identifiers.schema.json:8]:
```suggestion
    "trigger_id": {
```

File: gcn/notices/core/Identifiers.schema.json [hunk 5]
```diff
@@ -5,18 +5,25 @@
...
       "type": "string",
-      "description": "Trigger number or id number"
+      "description": "Trigger ID specific to the instrument (ex: Fermi bn230313485, Swift 1159327)"
```
Comment (jracusin) [inline gcn/notices/core/Identifiers.schema.json:10]:
```suggestion
      "description": "Instrument-specific trigger ID (ex: bn230313485 (Fermi), 1159327 (Swift)"
```

**Comment by jracusin** [comment]
Closing without merge.  See next PR with rearranged core schema.

**Pull request closed without merging**